docs/faq.md

```diff
@@ -390,6 +390,12 @@ See [How can I see the logs?](#how-can-i-see-the-logs).
 
 ### How can I see the logs?
 
+By default, we're configuring containers to use "none" as the logging driver, meaning that the output is simply discarded. By using 
```

Comment:
Log data is not discarded. We disable Docker's own logging because systemd saves logs for us in systemd-journald (`journalctl -fu matrix-synapse`).

Using a log-driver other than `none` means that all logs are stored twice, which is wasteful. Also, Docker doesn't seem to do any log rotation, so using something like json-file is bound to cause trouble, especially if you enable debug logging for Synapse (which will generate tons of logs).